web-api/src/embedding.rs

```diff
@@ -33,6 +35,16 @@ impl Default for Config {
     }
 }
 
+#[derive(Debug, Clone, Default, Deserialize, Serialize)]
+#[serde(default)]
+#[cfg_attr(test, serde(deny_unknown_fields))]
+pub(crate) struct Prefix {
+    /// Prefix prepended to search queries when embedding them.
+    pub(crate) query: Option<String>,
```

Comment:
I am not sure if we care to distinguish between prefix not given and an empty prefix. The end result will be the same.

Reply:
If you by default separate the prefix with a white-space from the rest it makes a difference.

Through we removed the default space separator so now it's fine.